x-pack/plugins/fleet/public/applications/fleet/app.tsx

```diff
@@ -213,11 +214,13 @@ export const FleetAppContext: React.FC<{
                     <ReactQueryDevtools initialIsOpen={false} />
                     <UIExtensionsContext.Provider value={extensions}>
                       <FleetStatusProvider defaultFleetStatus={fleetStatus}>
-                        <Router history={history}>
-                          <PackageInstallProvider startServices={startServices}>
-                            <FlyoutContextProvider>{children}</FlyoutContextProvider>
-                          </PackageInstallProvider>
-                        </Router>
+                        <SpaceSettingsContextProvider>
```

Comment:
I fetch the space settings here, instead of fetching it during the agent policy creation/edit, ..., in my opinion these settings should not change too much and it add a lot of extra complexity to add an async operation in the package policy agent policy editor.